index.html

```diff
@@ -2,222 +2,237 @@
 <html lang="en">
 
 <head>
-    <!-- Global site tag (gtag.js) - Google Analytics -->
-    <script async src="https://www.googletagmanager.com/gtag/js?id=G-CZ62YVL3TG"></script>
-    <script>
-      window.dataLayer = window.dataLayer || [];
-      function gtag(){dataLayer.push(arguments);}
-      gtag('js', new Date());
+  <meta charset="UTF-8">
+  <!-- Global site tag (gtag.js) - Google Analytics -->
+  <script async src="https://www.googletagmanager.com/gtag/js?id=G-CZ62YVL3TG"></script>
+  <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag(){dataLayer.push(arguments);}
+    gtag('js', new Date());
```

Comment:
- [ ] Kindly keep all the JavaScript code in JS files, not in the script tag. Let's follow the [Javascript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).

Reply:
@elyor-doniyorov Thanks for the suggestion but this is part of a previous Project.

Also, this is how it is supposed to be Since this is not my own Script, it's the script for Google Analytics and we are supposed to keep it at the top of the index.html file if you can please refer to this [link](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/exercises/google_analytics_exercise.md) and see the project requirements.